session/session.go

```diff
@@ -262,14 +287,6 @@ func (s *Session) onACK(surbID *[constants.SURBIDLength]byte, ciphertext []byte)
 		return nil
 	}
 
-	if msg.WithSURB && msg.IsDecoy {
-		_, ok := s.surbIDMap[*surbID]
-		if ok {
-			s.decrementDecoyLoopTally()
-			delete(s.surbIDMap, *surbID)
```

Comment:
where do the map entries get pruned?

Reply:
good question. i think i haven't yet written any code to prune the other map entries but should do so... here.